src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,87 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int CAPACITY = 12;
+    private static final int MAX_SIZE = 11;

```

Comment:
This constant is related to MAX_SIZE. Changing only one leads to an error (inconsistency).